modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -161,11 +161,20 @@
       end
 
       context 'pdf extra content verification' do
-        let(:extra_supplemental_claim) { create(:extra_supplemental_claim, created_at: '2021-02-03T14:15:16Z') }
+        include SchemaHelpers
+        let(:max_lengths) { find_max_lengths '../../schemas/v2/200995.json', 'maxLength' }
 
         it 'generates the expected pdf' do
-          generated_pdf = described_class.new(extra_supplemental_claim, version: 'V2').generate
-          expected_pdf = fixture_filepath('expected_200995_extra.pdf', version: 'v2')
+          sc = build(:extra_supplemental_claim)
+          sc.form_data = SchemaHelpers.supplemental_claim_w_fill(sc.form_data, max_lengths)
+          sc.save!
+          generated_pdf = described_class.new(sc, version: 'V2').generate
+          expected_pdf = fixture_filepath('expected_200995_maxlength.pdf', version: 'v2')
```

Comment:
The Ws in the telephone field are cut off, but I think it has something to do with printing the extension...which I don't think we actually do for domestic phone numbers. We'll have to touch base about this one.

Reply:
Update on my comment: I was wrong about the extension. We leave out extension for HLR because it just won't fit, but we include it for supplemental claim, so we'll definitely need to figure out how to deal with that field.

Reply:
Would the telephone ext fix be a separate ticket? or should I try to fix it in this one?

Reply:
Good call. Let's make a ticket so we can make a decision without feeling rushed.

Reply:
ticket created